src/CHANGELOG.tsx

```diff
@@ -35,6 +36,7 @@ import SpellLink from 'interface/SpellLink';
 
 // prettier-ignore
 export default [
+  change(date(2024, 8, 13), 'remove mindgame ref from hpriest', Liavre),
```

Comment:
Don't need an entry in the overall changelog - the one in Holy folder is enough.